### DISCUSSION TITLE
Replacing Native Anchor Tag with Link component

### DISCUSSION THREAD

**Discussion by gokul1630** [gatsbyjs/gatsby#36053] [Help]
Hi in my org, we have created our own design system with a few native anchor tags in the components. now the problem is, while using those components in gatsby, the whole page is getting reloaded while Navigating to a new page due to native anchor tags. I know that gatsby has a `Link `component but we can't implement this in our library, So is there any way to overcome this issue?

**Comment by jongwooo** [comment]


**Reply by gokul1630** [reply]
@jongwooo I guess you didn't understand my question.
